pkg/clusteragent/admission/mutate/agent_sidecar/providers.go

```diff
@@ -21,6 +23,10 @@ import (
 //                            //
 ////////////////////////////////
 
+const apmSocketDir = "/var/run/datadog"
```

Comment:
we should also mount the dogstatsd socket.
maybe we don't need to volume but one that container both socket